sqlalchemy_redshift/dialect.py

```
@@ -981,6 +981,191 @@ def _get_table_or_view_names(self, relkind, connection, schema=None, **kw):
+            if match is not None:
+                if issubclass(coltype._type_affinity, sqltypes.Integer):
+                    autoincrement = True
+                # the default is related to a Sequence
+                sch = schema
+                if "." not in match.group(2) and sch is not None:
+                    # unconditionally quote the schema name.  this could
+                    # later be enhanced to obey quoting rules /
+                    # "quote schema"
+                    default = (
+                        match.group(1)
+                        + ('"%s"' % sch)
```

Ruff (0.14.0)

Use format specifiers instead of percent format

Replace with format specifiers

(UP031)